src/main/java/com/xellitix/jenkins/plugintool/authentication/DefaultJenkinsApiTokenRetriever.java

```diff
@@ -0,0 +1,28 @@
+package com.xellitix.jenkins.plugintool.authentication;
+
+import com.xellitix.jenkins.plugintool.cli.command.FetchInstalledPluginsCommand;
+
+/**
+ * Default {@link JenkinsApiTokenRetriever} implementation.
+ *
+ * @author Garrett Ewens
+ */
+public class DefaultJenkinsApiTokenRetriever implements JenkinsApiTokenRetriever {
+
+  /**
+   * Gets the API token.
+   *
+   * @return The API token.
+   */
+  @Override
+  public String getApiToken(FetchInstalledPluginsCommand command) {
+    if (command.getApiToken().isPresent()) {
+      return command.getApiToken().get();
+    } else if (!System.getenv("JENKINS_API_TOKEN").isEmpty()
```

Comment:
You are calling `System.getenv("JENKINS_API_TOKEN")` 3 times. This is redundant. You can make the call once, save the result to a variable, then perform your checks and operations on the variable.